static/bidder-info/tpmn.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "https://gat.tpmn.io/ortb/pbs_bidder"
+maintainer:
+  email: "prebid@tpmn.io"
+modifyingVastXmlAllowed: true
+endpointCompression: GZIP
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
+    url: "https://gat.tpmn.io/sync/redirect?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
```

Comment:
The user sync URL is not setting the UID. Could you please check? Trying the following URL - 
```
https://gat.tpmn.io/sync/redirect?gdpr=&gdpr_consent=&us_privacy=&redir=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dtpmn%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24UID
```

Reply:
The macro function did not work normally, so it was fixed.
Now you can check that it works normally.

Thx!

Reply:
It is working now. Thanks!

Reply:
Good! Thank you for checking.